build/config/compiler/BUILD.gn

```diff
@@ -882,7 +882,7 @@ config("optimize") {
     cflags = [ "/O1" ] + common_optimize_on_cflags + [ "/Oi" ]
```

Comment:
This seems like an oversight on Windows that was inherited from old build rules. Can we file a bug separately to fix this please?